README.md

```diff
@@ -2,37 +2,108 @@
 
 A simple Docker & Drupal for development. This is all.
 
-### Apache + PHP 5.6
+## Nginx + PHP 5.6
 
-This branch is still on progress. For more stable, use the nginx branches.
+This branch is still in progress.
 
 What's included:
 
-  - PHP 5.6 (based on apache)
-  - Mariadb
+- Nginx
+- PHP-FPM 5.6
+- Mariadb
 
 Check the other branches of this repo for some more options.
 
 ### Official containers only
+
 All of this is based on officials containers, so you won't download any
 specific containers to use this project.
 
 ### Get started
+
 All your Drupal files should be placed into the folder "web".
 
+```sh
+mkdir web
+```
+
+Add the project domain to your local Hosts file, and point it to
+`127.0.100.100`.
+
+* Paste the following into `/etc/hosts` (`c:\Windows\System32\drivers\etc\hosts`
+  for Windows users):
+
+```
+127.0.100.100  drupal7.dev db.drupal7.dev
+```
+
+* Mac OSX users: Run the following to attach an unused IP to the lo0 interface.
+
+```sh
+sudo ifconfig lo0 alias 127.0.100.100/24
+```
+
 Run docker compose.
+
 ```sh
-$ docker-compose up -d
+docker-compose up -d
 ```
 
 Import your database or install your site.
+
 ```sh
-$  docker-compose run --rm php ./../vendor/bin/drush site-install
+docker-compose run --rm php ./../vendor/bin/drush site-install
 ```
 
 ### Database credentials
 This is for development !!
 
-* User: drupal
-* Password: drupal
-* Database: drupal
+- User: drupal
+- Password: drupal
+- Database: drupal
+- Host: db.drupal.dev
```

Comment:
"db" works, but only if Mac OSX is able to support static IP addresses. This solution has been working really well for me on my local, because I can execute Drush commands both inside and outside the php container. And I can still run multiple Docker-based sites side-by-side.

If we end up needing to use random ports as we discussed at our last weekly meeting, then we would have to give up on using this "hostname"-based solution for accessing the database, and then Drush could only be executed from inside the php container.